genai-perf/tests/test_openai_chat_converter.py

```diff
@@ -25,24 +25,54 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import pytest
+from typing import Any, List, Dict
 from genai_perf.inputs.converters import OpenAIChatCompletionsConverter
 from genai_perf.inputs.input_constants import ModelSelectionStrategy, OutputFormat
 from genai_perf.inputs.inputs_config import InputsConfig
+from genai_perf.inputs.retrievers.generic_dataset import GenericDataset, FileData, DataRow
 
 
 class TestOpenAIChatCompletionsConverter:
 
+    @staticmethod
+    def create_generic_dataset(rows: List[Dict[str, Any]]) -> GenericDataset:
+        def clean_text(row):
+            text = row.get("text", [])
+            if isinstance(text, list):
+                return [t for t in text if t]
```

Comment:
What are these cleanings actually doing? Looks like they're getting rid of `None`/`False` objects in a list? Why would those be there?

Reply:
Great catch, Matt. That's right. To be honest, I don't exactly remember. I know this solved a few issues to get the test to pass. It's possible that part of it was just passing single values vs lists to this function.

I can fix it for this PR or I can make a follow-up ticket to clean this up after merging. Do you have a preference?

Reply:
Thanks for explanation--follow up is fine 👍